gazelle.sql

```diff
@@ -30,6 +30,7 @@ CREATE TABLE `api_applications` (
   PRIMARY KEY (`ID`) USING BTREE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT;
 
+ALTER TABLE `api_applications` ADD UNIQUE(`UserID`);
```

Comment:
You should use `phinx` to do this, can refer `db/migrations/`

Reply:
@WhyNoobMaster69IsNotAvailable  this's the last, it still need you to adjust.

Reply:
i updated this already

Reply:
didn't push?

Reply:
has to do a pull and push
. should be done

Reply:
had to do a pull and push.

Reply:
just pushed now